ProgettoASD1718/src/base/Main.java

```diff
@@ -3,7 +3,7 @@
 public class Main {
 
 	public static void main(String[] args) {
-	
+		
```

Comment:
ricorda che il main può dare problemi di conflitti se usi per fare altri test in altri branch

Reply:
prima di pushare qualsiasi cosa elimino le modifiche del main, ma nel dubbio visto che lo tengo vuoto, lo eliminerò e lo ricreero ogni volta